book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(author, title)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
- [ ]  Kindly add the :rentals to the attr_accessor method so that the book class can read as well as change its value. :+1: 
- [ ] As part of the project requirements, kindly make sure that you create a method for adding rentals. The book itself should be sent as a parameter to create the new rental to help achieve the 'has-many' association.
Kindly check [this link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more information on that.